pkg/apis/infrastructure/v1alpha2/awsmachinetemplate_types.go

```diff
@@ -1,63 +0,0 @@
-/*
```

Comment:
Doh, I think the AWSMachineTemplate type should probably have been preserved.

Reply:
Oh yeah. Totally.